sprint-mission5/src/components/Allproducts.js

```diff
@@ -0,0 +1,72 @@
+import React, { useEffect, useState } from "react";
+import ItemCard from "./ItemCard";
+import { getProducts } from "../api";
+import "./allItems.css";
+
+function AllProducts() {
+  const [page, setPage] = useState("1");
+  const [orderBy, setOrderBy] = useState("recent");
+  const [pageSize, setPageSize] = useState("10");
+  const [itemList, setItemList] = useState([]);
+  const [keyword, setKeyword] = useState("");
+  const [totalPageNum, setTotalPageNum] = useState();
+
+  const handleNewestClick = () => setOrderBy("recent");
+
+  const handleFavoriteClick = () => setOrderBy("favorite");
```

Comment:
하나의 함수로 만들고 파라미터로 값을 받는 방법을 쓰면 중복이 줄 거 같아요!
```suggestion
  const handleOrderChangeClick = (chosenOrder) => setOrderBy(chosenOrder);
```